README.md

```diff
@@ -7,6 +7,7 @@
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
 [![Documentation Status](https://readthedocs.org/projects/visumorph/badge/?version=latest)](https://visumorph.readthedocs.io/en/latest/?badge=latest)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Python 3.12.0](https://img.shields.io/badge/python-3.7+-blue.svg)](https://www.python.org/downloads/)
```

Comment:
The alt text reads `3.12.0` but the badge is actually `3.7+`.

Also, it does not match our `pyproject.toml`, which states the supported version is from `3.9` and up. Please either change this badge, or change the pyproject.toml.

Reply:
@Marcony1